zerver/views/custom_profile_fields.py

```diff
@@ -59,14 +59,18 @@ def validate_field_name_and_hint(name: str, hint: str) -> None:
         raise JsonableError(error.message)
 
 
-def validate_custom_field_data(field_type: int, field_data: ProfileFieldData) -> None:
+def validate_custom_field_data(
+    field_type: int, field_data: Optional[ProfileFieldData] = None
```

Comment:
Can't we just keep the type here as `ProfileFieldData` by passing the original field data for the field in case it is not being changed.

Reply:
Updated at https://github.com/zulip/zulip/compare/c4c980bb5745e07ce93431b99e8d9b0a7cc9ef5d..0fce19c819eb24b5ed6365735703ea99173bc7dd